### PR TITLE
Ags encounter metadata

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/models/encounter/useEncounterFieldSchemas.js [hunk 1]
```diff
@@ -61,6 +57,11 @@ export default function useSightingFieldSchemas() {
         };
       });
 
+      speciesOptions.push({
+        label: intl.formatMessage({ id: 'UNKNOWN' }),
```
Comment (Atticus29) [inline src/models/encounter/useEncounterFieldSchemas.js:61]: Not clear to me why this should be hard-coded as unknown.

File: src/models/encounter/usePatchEncounter.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      );
+      dictionaryCopy.decimalLongitude = get(
+        dictionaryCopy,
+        ['gps', 1],
+        null,
+      );
+      delete dictionaryCopy.gps;
+    }
+
+    // delete dictionaryCopy.taxonomy;
```
Comment (Atticus29) [inline src/models/encounter/usePatchEncounter.js:27]: Why commented out? Just a bookmark for the future?

**Review by Atticus29 — COMMENTED** [review]
Generally, all looks good!

I'm curious why region isn't available in the encounter-level metadata edit:
Screen Shot 2021-12-03 at 2 32 47 PM
y

File: src/models/encounter/usePatchAgsEncounter.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  const [error, setError] = useState(null);
+  const [success, setSuccess] = useState(false);
+
+  const updateProperties = async (
+    assetGroupSightingId,
+    encounterId,
+    dictionary,
+  ) => {
+    const dictionaryCopy = { ...dictionary };
+    if ('gps' in dictionaryCopy) {
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/models/encounter/usePatchAgsEncounter.js:17]: yes!

File: src/models/encounter/useEncounterFieldSchemas.js (hunk 1 above)
Comment (brmscheiner) [reply src/models/encounter/useEncounterFieldSchemas.js:61]: it's just an option in the species dropdown to remove the existing species. so for example if they accidentally put zebra they can switch it back to unknown to remove the species property

File: src/models/encounter/usePatchEncounter.js (hunk 2 above)
Comment (brmscheiner) [reply src/models/encounter/usePatchEncounter.js:27]: i should remove this actually..

**Pull request merged**